### PR TITLE
Listed error codes in Readme

### PR DESCRIPTION
**Pull request by barbossa**
For convenience, I've decided to add all error codes to Readme file

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,6 +34,22 @@ Since version 2.6 we introduce new ``--isort-show-traceback`` option.
 It is used to show verbose multi-line output from ``isort``.
 By default it is turned off.
 
+Error codes
+-------
```
Comment (gforcada) [inline README.rst]: Could you make the underline match the title?

**Comment by gforcada** [comment]
Thanks for the pull request 😃

Seems that Travis is no happy though 🤔

File: README.rst (hunk 1 above)
Comment (barbossa) [reply README.rst]: @gforcada Sorry, did not understand you. Underline match the title?

Comment (barbossa) [reply README.rst]: What you mean?

Comment (gforcada) [reply README.rst]: I mean that the number of dashes match the length of the title above the dashes, i.e.:
```
Error codes
-----------
```

That would produce an RST syntax error when releasing to pypi for example.

Comment (barbossa) [reply README.rst]: Oh, sure. Yes, I will do it.

Comment (barbossa) [reply README.rst]: @gforcada fixed.

**Comment by barbossa** [comment]
@gforcada hey,
I've fixed error with title. Can you merge?

**Pull request merged**